dev/com.ibm.ws.messaging.comms.client/src/com/ibm/ws/sib/jfapchannel/framework/NetworkConnectionFactory.java

```diff
@@ -42,10 +40,11 @@ public interface NetworkConnectionFactory
      * @throws FrameworkException if the connection cannot be created.
      */
     NetworkConnection createConnection() throws FrameworkException;
-
+    
     /**
-     * 
-     * @return an VirtualConnectionFactory
+     * Destroys the connection object stored
+     * @return if connection was successfully destroyed
```

Comment:
What happens if the connection was not successfully destroyed?
Maybe add an `@throws` line.

Reply:
Added a throws line for this and more clarification. In this case, it's not a specific connection but the object necessary to start up connections for a factory. For chfw there is a virtual connection factory for managing connections which has its own destroy method but for Netty our bootstrap defines how connections are started so the description was not much accurate. Updated to match this.